sys/txs-compiler/src/TorXakis/Parser/ValExprDecl.hs

```diff
@@ -126,7 +126,7 @@ txsITEP = do
 txsBopSymbolP :: TxsParser Text
 txsBopSymbolP = try $ T.pack <$> do
     op <- txsLexeme (many1 txsSpecialCOp)
-    if op `elem` [">->", "|", "||", "|||", ">>>", "[>>", "[><", "##", "<-"]
+    if op `elem` [">->", "|", "||", "|||", ">>>", "[>>", "[><", "##", "<-", "->"]
```

Comment:
In CNECTDEF, also "<-" and "->" are used.
@dnadales did you also check it there? E.g. with something like:
```
ENCODE Channel ? a ? b -> ! toString(a) ++ " " ++ toString(b)
```